importer/src/cli.ts

```diff
@@ -0,0 +1,21 @@
+import { program } from 'commander';
+import { run } from './cli-run.js';
+import { IMPORTER_VERSION } from './version.js';
+
+program
+  .name('xata-importer')
+  .description('Importer utility for xata.io')
+  .version(IMPORTER_VERSION)
+  .argument('<file>', 'File containing the data to import. Pass "-" to pipe content instead.')
```

Comment:
I would add another option for ``delimiter`` and probably attempt to make it "smart".

Although CSV stands for comma-separated-values, the delimiter could be anything. Most popular are semicolons, tabs, spaces, etc...

Reply:
Good idea.

Question about one use case for that. How could we specify tab delimiters with flags in the CLI? 🤔

Reply:
What about "\t"